src/components/HeroHeader/HeroHeader.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { css } from '@linaria/core';
+
+export const HeroHeader = () => {
+    const heroImageStyle = css`
```

Comment:
can we please move these `linaria` components out of the `HeroHeader` component?

Reply:
Moved them to its own file, called `CSS.HeroHeader.tsx`.